src/h5web/visualizations/nexus/utils.ts

```diff
@@ -0,0 +1,34 @@
+import {
+  HDF5Group,
+  HDF5Metadata,
+  HDF5Dataset,
+  HDF5Value,
+} from '../../providers/models';
+import { getEntity, isDataset } from '../../providers/utils';
+import { NxAttribute } from './models';
+
+export function isNxDataGroup(group: HDF5Group): boolean {
+  return !!group.attributes?.find(({ value }) => value === 'NXdata');
+}
+
+export function getAttributeValue(
+  entity: HDF5Dataset | HDF5Group,
+  attributeName: NxAttribute
```

Comment:
The typing means `getAttributeValue` cannot be used outside a NeXuS context.

Not a huge deal for now as we don't use it elsewhere but worth keeping in mind.

Reply:
That's kind of why I had removed the enum and left a simple `string`. I figured that we were just as likely to make a typo in the call to `getAttributeValue` as we were to make a typo in the enum or the union type. Either way, you would notice the typo quite quickly since the visualization would just not work 😄. Anyway, we can see what happens later on. We can rename the union type to include other HDF5 attributes, rename the function `getNxAttributeValue`, go back to a simple `string`, etc. plenty of options.